source/reference/collection-level-filtering.txt

```diff
@@ -17,6 +17,62 @@ Filtered Sync
 
 .. include:: /includes/api/facts/filtering-intro.rst
 
+Starting in 1.1, ``mongosync`` supports inclusion filters to specify which
+databases and collections to include in sync.  Starting in 1.6, ``mongosync``
+also supports exclusion filters and regular expressions.
+
+- With inclusion filters, ``mongosync`` syncs matching
+  databases and collections.
+- With exclusion filters, ``mongosync`` syncs all databases
+  and collections, except for those that match the filters.
+- With both inclusion and exclusion filters, ``mongosync`` only syncs
```

Comment:
This makes it sound like the exclusion filter is ignored. Can we phrase this to make it clear that the exclusion filter overrides whatever is in the inclusion filter?

Reply:
I agree. Maybe something along the lines:
With both inclusion and exclusion filters, ``mongosync`` only syncs databases and collections that match the inclusion filters and are not excluded by the exclusion filters?

Reply:
Fixed